### PR TITLE
Apollo: Update docs to show new "variant" tag support

### PR DESCRIPTION
**Pull request by sachindshinde**
### What does this PR do?

The changes are primarily:
- update docs to show support for a new tag available on submitted metrics (the "variant" tag)
- change the integration name from "Apollo Engine" to "Apollo"
- many docs typo fixes/rewordings

### Motivation

We maintain software that submits metrics to Datadog via the HTTP API, and we recently made an update to that software so that it tags submitted metrics with a new tag (with tag key "variant"). In updating these docs to reflect that, we noticed a lot of things were out-of-date (including the display name of "Apollo Engine", as that's been rebranded), and decided to fix them as well.

Let me know if you'd rather have this PR split into smaller PRs, and I can rebase as needed.

Also I'm guessing I need to open a PR/issue in the [documentation repo](https://github.com/DataDog/documentation/), but I'm not sure what needs changing.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

I don't actually know how difficult it is to change an integration's name from Datadog's end, let me know if there's anything I can do to help there. (I've structured this PR so the last commit is the one that changes the directory from `apollo_engine` to `apollo` and updates the `manifest.json` name.)

### CODE REVIEW

File: apollo/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+All of the metrics are also tagged with the Graph Manager graph ID as `service:<graph-id>` and the variant name as `variant:<variant-name>`, so multiple graphs from Graph Manager can send data to the same Datadog account. If you have not set a variant name, then "current" will be used.
+
+If you're reporting metrics to Graph Manager through the Engine proxy, Datadog will merge your statistics across multiple instances of the proxy (per-host metrics are not available). Just like in the Graph Manager UI, each operation inside a query batch is counted individually.
+
+## Setup
+
+### Configuration
+
+Getting set up with the Apollo Datadog integration is as simple as providing a Datadog API key to Graph Manager. There's no further configuration required!
```
Comment (sarina-dd) [inline apollo/README.md]:
```suggestion
Getting set up with the Apollo Datadog integration is as simple as providing a Datadog API key to Graph Manager. There's no further configuration required.
```

File: apollo/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+1. Go to the [Datadog integrations page][2] and click on the Apollo tile. Go to the _Configuration_ tab, scroll to the bottom, and press **Install Integration**.
```
Comment (sarina-dd) [inline apollo/README.md]:
```suggestion
1. Go to the [Datadog integrations page][2] and click on the Apollo tile. Go to the **Configuration** tab, scroll to the bottom, and press **Install Integration**.
```

**Comment by sachindshinde** [comment]
Thanks for the feedback, @sarina-dd ! I've added your changes to the PR.

File: apollo/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# CHANGELOG - Apollo Integration
+
+1.0.0
+==================
+### Changes
+
+* [FEATURE] Initial Apollo Integration Tile.
+
+1.1.0
```
Comment (hithwen) [inline apollo/CHANGELOG.md]: Please put newer version on top of the file

File: apollo/manifest.json [hunk 4]
```diff
@@ -1,19 +1,21 @@
...
-    "linux"
+    "linux",
+    "mac_os",
+    "windows"
   ],
-  "manifest_version": "1.0.0",
-  "maintainer": "martijnwalraven@apollographql.com",
-  "display_name": "Apollo Engine",
+  "manifest_version": "1.1.0",
+  "maintainer": "sachin@apollographql.com",
```
Comment (hithwen) [inline apollo/manifest.json]: please add to `.github/CODEOWNERS`

**Comment by sachindshinde** [comment]
Thanks for the feedback, @hithwen ! I've added your changes to the PR.

**Pull request merged**